tests/tools/RxTxApp/src/tx_st30p_app.c

```diff
@@ -196,12 +213,29 @@ static int app_tx_st30p_init(struct st_app_context* ctx, st_json_st30p_session_t
   ops.channel = st30p ? st30p->info.audio_channel : 2;
   ops.sampling = st30p ? st30p->info.audio_sampling : ST30_SAMPLING_48K;
   ops.ptime = st30p ? st30p->info.audio_ptime : ST30_PTIME_1MS;
+
+  if (st30p && st30p->user_pacing) {
+    s->packet_time = st30_get_packet_time(ops.ptime);
+  } else {
+    s->packet_time = 10 * NS_PER_MS; /* default 10ms */
```

Comment:
Please use ST_APP_USER_PACING_DEFAULT_OFFSET if possible

Reply:
This is diffrent than offset 

Offset is value we add to the base time to give MTL time to process the packet before sending, 
Here we are setting the size of the packet we want for the St30 pipeline mode framebuffers 

But it still holds true that this should be set by a define instead of some magic number 
Good comment I'm changing it to define

Reply:
Changed this in 33e16e4839edb04aeb7b43ec93c57cecc415d87d
by adding 
ST_APP_TX_ST30P_DEFAULT_PACKET_TIME 

makes more sense IMO